README.MD

```diff
@@ -4,15 +4,105 @@ This plugin is only needed if you want to host a public SRV mode proxy instance.
 
 ## SRV ViaProxy
 ### Running ViaProxy in SRV mode
-To use the SRV mode in ViaProxy you have to start it through the command line.\
-The command line arguments are as follows:
-````bash
-java -jar ViaProxy.jar -ba 0.0.0.0 -bp <bind port> -srv -oam_auth -a0 -v1.8.x
-````
+> [!CAUTION]
+> This Option is meaningless because there is no proper authentication method for 1.19.4+ as OpenAuthMod discontinued and because ViaProxy was open source for 1 year is easier to use on client without their help. 
+use this option if you want to.
+
+To use the SRV mode in ViaProxy you have to start it through the config.\
+The config:
+```
+# ViaProxy configuration file
```

Comment:
Maybe instead of pasting the entire ViaProxy config which can change any time, only the relevant options should be here (maybe even without comments?)

Reply:
ok, i am changing it instantly.